src/main/java/ch/njol/skript/expressions/LitPi.java

```diff
@@ -0,0 +1,58 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleLiteral;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Pi")
+@Description("Returns the mathematical constant pi.")
+@Examples("set {_tau} to pi * 2")
+@Since("INSERT VERSION")
+public class LitPi extends SimpleLiteral<Number> {
+
+	static {
+		Skript.registerExpression(LitPi.class, Number.class, ExpressionType.SIMPLE, "(pi|π)");
+	}
+
+	public LitPi() {
+		super(Math.PI, false);
+	}
+
+	@Override
+	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final SkriptParser.ParseResult parseResult) {
```

Comment:
Import `ParseResult` directly (not statically)
We don't use `final` modifier on methods params
```suggestion
	public boolean init( Expression<?>[] exprs,  int matchedPattern,  Kleenean isDelayed, ParseResult parseResult) {
```